_sass/leaderboard.scss

```
@@ -61,6 +61,80 @@ body.dark-mode .leaderboard-filter::placeholder {
+    position: relative;
+    display: flex;
+    justify-content: flex-end;
+    margin: 0 0 1.5rem;
+}
+
+.scoring-info-toggle {
+    display: inline-flex;
+    align-items: center;
+    gap: 0.4rem;
+
+    padding: 0.5rem 1rem;
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)